tests/integration/tests/namespace/test-namespace-overview.spec.js

```diff
@@ -33,13 +33,30 @@ context(
         .scrollIntoView()
         .should('be.visible');
 
-      cy.contains('Memory Requests').should('be.visible');
-
-      cy.contains('Memory Limits').should('be.visible');
-
       cy.contains('Events')
         .scrollIntoView()
         .should('be.visible');
     });
+
+    it('checks the visibility of charts', () => {
```

Comment:
Why this test checks errors and loading?
I think it should check if charts are available and visible.